lib/netsuite/records/note.rb

```diff
@@ -0,0 +1,36 @@
+module NetSuite
+  module Records
+    class Note
+      OUTGOING = '_outgoing'
+      INCOMING = '_incoming'
```

Comment:
I'm definitely a fan of avoiding magic strings. However, right now we don't include magic strings in the gem for other aspects of the API. It's best to omit these from the gem for now until we create a generic solution for handling magic strings.